### ISSUE TITLE
Points only

### ISSUE THREAD

**Issue by gguandf** [mapbox/tippecanoe#948]
First - thanks for this powerful library.

I'm trying to create an mbtile-file with tippecanoe.
The base is a geojson with 64800 points.
Each point has coordinates and a value.

like: 

`{
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [ 6.0,  -89.5 ]
      },
      "properties": {
        "value": -53.0
      }
},`

Question 1: can this work?
Question2: can tippecanoe deal with points only?

The problem: When the mbtile-file is loaded into SQLite-Viewer, metadata and tiles (binary) are shown but no images!?
It is similar with tileserver-php-master - no images are loaded.

Any ideas what i'm doing wrong?
The goal is to show all GeoJson points in a leaflet map.
But with tiles which are loaded when zooming and scrolling - and not with one giant geojsons file.

**Comment by e-n-f** [comment]
Thanks! Tippecanoe development is now taking place in https://github.com/felt/tippecanoe.

Features like these should work with Tippecanoe. Tippecanoe can deal with points, LineStrings, and Polygons.

Tippecanoe creates vector tiles; it is necessary to use a renderer to see the output as images. You can for instance use [Protomaps](https://github.com/protomaps/protomaps.js) or [Mapbox GL](https://docs.mapbox.com/mapbox-gl-js/guides/).

**Comment by gguandf** [comment]
Hi, thanks for your answer.

I'm wondering how to display the *.mbtiles-files in a leaflet-map.
I allready tried TileLayer.MBTiles.js!

If I view the *.mbtiles-file in SQLite Viewer the format is 'pbf' !?!
My leaflet-map is empty (in each zoom-level) - only the background-map (osm) is shown.

No JS-errors are returned.

kind regards.

**Comment by e-n-f** [comment]
I am no expert on Leaflet, but apparently there is a [plugin](https://github.com/SpatialServer/Leaflet.MapboxVectorTile) to display vector tiles.

The "pbf" refers to the format defined [here](https://github.com/mapbox/vector-tile-spec), which is based upon Protocol Buffers.

**Comment by gguandf** [comment]
Hello Erica, thanks again.

You steered me in the right direction!

With 'Leaflet.VectorGrid.bundled.js' and [Tile-Sever](https://github.com/maptiler/tileserver-php)
now I can display the *.mbtiles as '{z}/{x}/{y}.pbf' on my leaflet-map.

Now I have to style the vector tiles - this can be done with the 'vectorTileLayerStyles' option.
I.e. each layer can have its own style variants.

The goal is an mbtiles file with several layers.
Is this possible with tippecanoe?

Kind regards.

**Comment by e-n-f** [comment]
Yes, if you supply multiple `.json` source files, each will be placed in its own layer, or you can use the `-L` option to associate sources with layers, or you can use the `"tippecanoe": { "layer": name }` field in a feature to specify its layer.

**Comment by gguandf** [comment]
OK. I understand that I can put different JSON source files in different layers.
But can I only use one JSON file and assign different layer names based on the attribute values?

eg:
Values from 4 to 8 Layer-Name-X
Values from 8 to 12 Layer-Name-Y

eg.
```
{
   "type": "FeatureCollection",
   "features": [
     {
       "type": "feature",
       "geometry": {
         "type": "Point",
         "coordinates": [
           150.0,
           90.0
         ]
       },
       "properties": {
         "value": 8.0
       }
     },
     {
       "type": "feature",
       "geometry": {
         "type": "Point",
         "coordinates": [
           151.0,
           90.0
         ]
       },
       "properties": {
         "value": 10.0
       }
     }
]
```

**Comment by e-n-f** [comment]
Tippecanoe does not have the ability to filter layers by attribute value. You would have do something like:

```
{"type":"Feature","tippecanoe":{"layer":"one"},"geometry":{"type":"Point","coordinates":[150,45]},"properties":{"value":8}}
{"type":"Feature","tippecanoe":{"layer":"two"},"geometry":{"type":"Point","coordinates":[151,45]},"properties":{"value":10}}
```

to explicitly tag each feature with its layer.

**Comment by e-n-f** [comment]
I wrote the latest revision of the mbtiles spec, so I think TileServer is wrong, not tippecanoe, but if it doesn't like the `generator_options`, you can comment out this block of code and not include it in the metadata:

https://github.com/felt/tippecanoe/blob/182093bdc78d0a0adc65d24c9cbbe93eecaa38cf/mbtiles.cpp#L450-L457
